pages/index.md

```diff
@@ -18,164 +16,43 @@ Through the DSNP, we hope mitigate and ideally solve these challenges in the way
 
 ## How to Read This Specification
 
-The specification is divided into several modules.
-Each module describes a mostly self-contained aspect of the DSNP and includes documents detailing the function and intended use the module.
+There are three core specifications currently.
 
-Proposals for some of these future extensions can be found in the [Draft Specifications](#draft-specifications) section and on [GitHub](https://github.com/LibertyDSNP/spec/labels/enhancement),
-however these proposals should be regarded with caution as they are largely incomplete and may be subject to change or deletion without notice.
+| Name | Version | Description |
+| --- | --- | --- |
+| [DSNP](/DSNP/Overview) | 1.0.0 | The implementation agnostic DSNP specification |
```

Comment:
The patch version here refers to the version number of the DSNP spec itself, right?  If so, the spec itself isn't changing just because the web site on which the spec is presented has changed.

We could have a separate version number for the web site presentation if we wanted to, but I'm not sure that would serve any practical purpose.  Instead, you could, e.g., show a last updated date on the site, in the footer or something, so that people who are watching closely can know that things might have been moved around since the last time they were here (even though the spec *per se* hasn't changed -- i.e., no one's *implementation* of DSNP would be affected by merge web site rearrangement, whereas an implementation might indeed be affected by an actual spec change).